pkg/analytics/README.md

```diff
@@ -0,0 +1,118 @@
+# What Data is shared by users of Bacalhau?
```

Comment:
@seanmtracey @aronchick is here is README on the data being collected, along with steps to opt out. Feedback welcome.